README.md

```diff
@@ -38,114 +46,106 @@ The league establishes a map pool for each season. The map pool for Season 1 of
 
-Teams select timezones to play in. By default, timezones for all teams are Central Time.
+### Check-in
 
-- **Select Eastern Time** if you want to play earlier in the evening. (EST, INDIANA)
-- **Select Central Time** if you want to play in the middle of the evening.
-- **Select Pacific Time** if you want to play later in the evening. (PST, MST, ALASKA, HAWAII, ARIZONA)
+Four or five players from your team must check in on the match page for each round before the round can start. **Matches cannot be reported properly if players do not check in, so be sure to remember to do this before *each* round!**
 
-## Divisions
+### Tardy Policy
 
-### Selecting a Division
+Teams who are not ready to play 10 minutes after the scheduled start time can be marked as no-shows. However, if you are waiting on and in contact with your opponents, please give them some sportsmanlike leeway. Likewise, please respect your opponents' and the caster's time. You can report no shows on the match page within your team's management view.
 
-Each league consists of many divisions, ordered by competitiveness, with Division 1 having the highest level of competition. Choose a division not based on indivudual skill, but by skill as a team. Currently, there is no way to change divisions mid-season.
+### Casters
 
-**Players who have achieved Master or higher in any Hero or Team League season are ineligible to compete in Division 3.** Aside from this restriction, all divisions are open to anyone.
+If a match is being casted, the caster(s) should be added to the lobby and set to "referee" status. 
 
-### Standings
-      
-Division standings are ordered by points. Teams earn points only in their weekly division match. Wins will reward +3 points while losses will not reward any points.
+### Substitutions
 
-### Season Rating
+Teams may sub in one non-rostered player for games, so long as that player is eligible to play in that league (see [Divisions](#divisions) and [Playing in Multiple Leagues](#playing-in-multiple-leagues-teams)). Substitutions can occur in between rounds - player who plays in any round is not required to play in any other rounds in the series. However, once drafting has begun, the roster is locked for that round.
 
-During season play, team's will acquire a seasonal rating. This can be viewed in the standing of the division. The rating represents how a team is doing relative to other teams. Higher ratings show higher performance. Chair League uses this rating to match your team against opponents of similar skill level within your division. Rating gained or loss depends on the outcome of the weekly match and the magnitude is related to the rating difference between the two teams.
+### Map Selection / Draft Order
 
-## Tournaments
+The Home team, as designated by the schedule, is responsible for selecting the map and hosting the lobby for the first match. The Away team will thus have the first pick in the draft. This then alternates each round, as such:
```

Comment:
The addition of the table makes the "as such" awkward.  Would say "This then alternates each round.  As such:"  (aka period instead of comma)